cadence/freshmint-claim-sale/FreshmintClaimSale.cdc

```diff
@@ -392,6 +437,7 @@ pub contract FreshmintClaimSale {
             receiverPath: receiverPath,
             paymentReceiver: paymentReceiver,
             price: price,
+            claimLimit: claimLimit,
```

Comment:
Hmm question, could someone keep a reference to allowList and change it at anytime? Do we need to make a copy of it somehow?

Reply:
Yeah, that's the intended usage. `allowlist` is a capability that points to the actual allowlist resource, which lives elsewhere in storage. 

This allows a user to create the allowlist before creating the sale. It also lets them use the same allowlist across multiple sales. We assume that the owner of the sale is also the owner of the allowlist and therefore the capability should be private (only accessible by the owner).